src/prismic/Document.cs

```diff
@@ -95,6 +95,9 @@ public class Document: WithFragments
 				return fragments;
 			}
 			var type = (string)json["type"];
+			if (type == null)
+                		return fragments;
```

Comment:
Could you tell me when we encounter this case?

Reply:
Seems that this is not encountered, at least I cannot reproduce the situation. So I removed these lines of code in the new commit